pkg/virt-api/webhooks/validating-webhook/admitters/vmi-create-admitter_test.go

```diff
@@ -394,54 +394,52 @@ var _ = Describe("Validating VMICreate Admitter", func() {
 	Context("with VirtualMachineInstance metadata", func() {
 		DescribeTable(
 			"Should allow VMI creation with kubevirt.io/ labels only for kubevirt service accounts",
-			func(vmiLabels map[string]string, userAccount string, positive bool) {
-				vmi := api.NewMinimalVMI("testvmi")
-				vmi.Labels = vmiLabels
+			func(labelKey, labelValue, userAccount string) {
```

Comment:
Please consider taking the labels as a map, as it makes the overall test shorter and groups the key-value pairs nicely.

Same bellow.

Reply:
`WithLabel` / `WithAnnotation` take a key and value as individual strings.
Did you mean something like this?
```go
labelOptions := make([]libvmi.Option, 0)
for label, value := range labels {
	labelOptions = append(labelOptions, libvmi.WithLabel(label, value))
}
vmi := newBaseVmi(labelOptions...)
```

I could also introduce a wrapper like:
```go
func withLabels(labels map[string]string) []libvmi.Option {
	options := make([]libvmi.Option, 0)
	for label, value := range labels {
		options = append(options, libvmi.WithLabel(label, value))
	}
	return options
}
```

But I think it's overkill:
```bash
$ egrep "WithLabel|WithAnnotation" pkg/virt-api/webhooks/validating-webhook/admitters/vmi-create-admitter_test.go | wc -l
4
```

WDYT?